doc/api/child_process.md

```diff
@@ -405,6 +405,7 @@ changes:
     `'/bin/sh'` on UNIX, and `process.env.ComSpec` on Windows. A different
     shell can be specified as a string. See [Shell Requirements][] and
     [Default Windows Shell][]. **Default:** `false` (no shell).
+  * `windowsVerbatimArguments` {Boolean} No quoting or escaping of arguments is done on Windows. Ignored on Unix.
```

Comment:
nit: long line. need to wrap at <= 80 chars

Reply:
updated and pushed.